tldw_Server_API/app/core/Ingestion_Media_Processing/visual_ingestion.py

```
@@ -0,0 +1,154 @@
+                        frame_index=None,
+                        timestamp_seconds=None,
+                        thumbnail_path=None,
+                        extra_metadata=json.dumps(extra),
+                    )
+                    created += 1
+                except (ValueError, TypeError, KeyError, json.JSONDecodeError) as det_err:
+                    logger.debug(f"visual_ingestion: skipping detection due to data error: {det_err}")
+                    continue
+                except Exception as det_err:
+                    logger.exception(
+                        "visual_ingestion: unexpected error while persisting detection",  # noqa: TRY401
```

Ruff (0.14.6)

Unused `noqa` directive (unused: `TRY401`)

Remove unused `noqa` directive

(RUF100)